hosting/src/models/event.ts

```diff
@@ -0,0 +1,32 @@
+import dayjs, { Dayjs } from 'dayjs'
```

Comment:
Why don't we put this file in an `events` package?

Reply:
Had a god at this... I really like it

![Screen Shot 2021-04-22 at 11 00 46](https://user-images.githubusercontent.com/31621095/115653792-09b25b80-a35a-11eb-9492-234c58d0f786.png)

Note: Once the other packages are migrated, `/views` directory would disappear completely, leaving only functional packages: `people`, `home`, etc.

Reply:
Ran with this a bit, and ended up here:

![Screen Shot 2021-04-22 at 11 56 08](https://user-images.githubusercontent.com/31621095/115657837-c52abe00-a361-11eb-842c-77c07a1c5386.png)

Reply:
DONE